src/python/pants/engine/internals/scheduler.py

```diff
@@ -175,11 +175,6 @@ def __init__(
         )
         remoting_options = PyRemotingOptions(
             execution_enable=execution_options.remote_execution,
-            store_address=execution_options.remote_store_address,
```

Comment:
What changed in the upgrade to prompt reordering parameters (here and several other places)?

Reply:
A warning/error about optional parameters coming before non-optional parameters.